multipatch_analysis/qc.py

```diff
@@ -74,43 +91,59 @@ def pulse_response_qc_pass(post_rec, window, n_spikes, adjacent_pulses):
         Whether this pulse-response passes QC for detecting excitatory connections
     in_qc_pass : bool
         Whether this pulse-response passes QC for detecting inhibitory connections
+    failures : dict
+        QC failures for ex and in
     """
+    
+    failures = {'ex': [], 'in': []}
+
     # Require the postsynaptic recording to pass basic QC
-    if recording_qc_pass(post_rec) is False:
-        return False, False
+    recording_pass_qc, _ = recording_qc_pass(post_rec)
+    if recording_pass_qc is False:
+        [failures[k].append('postsynaptic recording failed QC') for k in failures.keys()]
```

Comment:
copy the failures reported from `recording_qc_pass` rather than adding the generic `'postsynaptic recording failed QC'` message

Reply:
yeah I thought about that too actually, so I'll do that. Only reason I hadn't was I thought that one could just go to the recording and see the message